tutorials/image/cifar10_estimator/cifar10_main.py

```diff
@@ -317,9 +317,10 @@ def _resnet_model_fn(features, labels, mode):
         num_batches_per_epoch * x
         for x in np.array([82, 123, 300], dtype=np.int64)
     ]
-    staged_lr = [FLAGS.learning_rate * x 
-        for x in [1, 0.1, 0.01, 0.002]]
-    
+    staged_lr = [
+        FLAGS.learning_rate * x for x in [1, 0.1, 0.01, 0.002]
+    ]
+
```

Comment:
You can just make the whole thing one line like this:

`staged_lr = [FLAGS.learning_rate * x for x in [1, 0.1, 0.01, 0.002]]`